src/main/java/com/netgrif/application/engine/pdf/generator/service/renderer/I18nDividerFieldRenderer.java

```diff
@@ -0,0 +1,53 @@
+package com.netgrif.application.engine.pdf.generator.service.renderer;
+
+import com.netgrif.application.engine.pdf.generator.domain.PdfField;
+import com.netgrif.application.engine.pdf.generator.domain.PdfTextField;
+import com.netgrif.application.engine.pdf.generator.service.fieldbuilder.FieldBuilder;
+import org.apache.pdfbox.pdmodel.font.PDType0Font;
+
+import java.awt.*;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class I18nDividerFieldRenderer extends FieldRenderer {
+
+    public void setFieldParams(PdfField field) {
+        helperField = new PdfTextField(field.getFieldId(), field.getLabel(), field.getValues(), field.getType(),
+                resource.getBaseX() + field.getX(), resource.getBaseY() - field.getBottomY(), field.getWidth(), field.getHeight(), resource);
+    }
+
+    @Override
+    public int renderLabel(PdfField field) throws IOException {
+        return 0;
+    }
+
+    @Override
+    public void renderValue(PdfField field, int lineCounter) throws IOException {
+        setFieldParams(field);
+        renderValue(helperField, resource.getLabelFont(), resource.getFontGroupSize(), colorDataGroupLabel);
+    }
+
+    private void renderValue(PdfField field, PDType0Font font, int fontSize, Color colorLabel) throws IOException {
+        float textWidth = getTextWidth(Collections.singletonList(field.getLabel()), font, fontSize, resource);
+        int maxLineSize = getMaxLabelLineSize(field.getWidth(), fontSize);
+        List<String> multiLineText = new ArrayList<>() {{
```

Comment:
bug marked by sonar.
If `field.getValues()` returns a `Collection`, then it should be possible to just call `new ArrayList<>(field.getValues())` to achieve the same result

Reply:
corrected